modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -9,117 +9,7 @@ import (
 	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
 )
 
-func (suite *KeeperTestSuite) TestEscrowPacketFee() {
-	var (
-		err        error
-		refundAcc  sdk.AccAddress
-		ackFee     sdk.Coins
-		receiveFee sdk.Coins
-		timeoutFee sdk.Coins
-		packetID   channeltypes.PacketId
-	)
-
-	testCases := []struct {
-		name     string
-		malleate func()
-		expPass  bool
-	}{
-		{
-			"success", func() {}, true,
-		},
-		{
-			"success with existing packet fee", func() {
-				fee := types.Fee{
-					RecvFee:    receiveFee,
-					AckFee:     ackFee,
-					TimeoutFee: timeoutFee,
-				}
-
-				packetFee := types.NewPacketFee(fee, refundAcc.String(), []string{})
-				feesInEscrow := types.NewPacketFees([]types.PacketFee{packetFee})
-
-				suite.chainA.GetSimApp().IBCFeeKeeper.SetFeesInEscrow(suite.chainA.GetContext(), packetID, feesInEscrow)
-			}, true,
-		},
-		{
-			"fee not enabled on this channel", func() {
-				packetID.ChannelId = "disabled_channel"
-			}, false,
-		},
-		{
-			"refundAcc does not exist", func() {
-				// this acc does not exist on chainA
-				refundAcc = suite.chainB.SenderAccount.GetAddress()
-			}, false,
-		},
-		{
-			"ackFee balance not found", func() {
-				ackFee = invalidCoins
-			}, false,
-		},
-		{
-			"receive balance not found", func() {
-				receiveFee = invalidCoins
-			}, false,
-		},
-		{
-			"timeout balance not found", func() {
-				timeoutFee = invalidCoins
-			}, false,
-		},
-	}
-
-	for _, tc := range testCases {
-		tc := tc
-
-		suite.Run(tc.name, func() {
-			suite.SetupTest()                   // reset
-			suite.coordinator.Setup(suite.path) // setup channel
-
-			// setup
-			refundAcc = suite.chainA.SenderAccount.GetAddress()
-			receiveFee = defaultRecvFee
-			ackFee = defaultAckFee
-			timeoutFee = defaultTimeoutFee
-			packetID = channeltypes.NewPacketId(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, uint64(1))
-
-			tc.malleate()
-			fee := types.Fee{
-				RecvFee:    receiveFee,
-				AckFee:     ackFee,
-				TimeoutFee: timeoutFee,
-			}
-			packetFee := types.NewPacketFee(fee, refundAcc.String(), []string{})
-
-			// refundAcc balance before escrow
-			originalBal := suite.chainA.GetSimApp().BankKeeper.GetBalance(suite.chainA.GetContext(), refundAcc, sdk.DefaultBondDenom)
-
-			// escrow the packet fee
-			err = suite.chainA.GetSimApp().IBCFeeKeeper.EscrowPacketFee(suite.chainA.GetContext(), packetID, packetFee)
-
-			if tc.expPass {
-				feesInEscrow, found := suite.chainA.GetSimApp().IBCFeeKeeper.GetFeesInEscrow(suite.chainA.GetContext(), packetID)
-				suite.Require().True(found)
-				// check if the escrowed fee is set in state
-				suite.Require().True(feesInEscrow.PacketFees[0].Fee.AckFee.IsEqual(fee.AckFee))
```

Comment:
Should we add these checks somewhere also?

Reply:
Done